src/components/Tool/Appeal/Modals/PledgeModal/PledgeModal.tsx

```diff
@@ -232,74 +232,144 @@ export const PledgeModal: React.FC<PledgeModalProps> = ({
               </Grid>
 
               {/* Amount and Currency Row */}
-              <Grid container item xs={12} spacing={1}>
-                <Grid item xs={isMobile ? 12 : 8}>
-                  <FormControl
-                    size="small"
-                    fullWidth
-                    error={!!errors.amount && touched.amount}
-                  >
-                    <LogFormLabel
-                      htmlFor="amount-input"
-                      id="amount-input-label"
-                      required
+              <Grid container item xs={12} spacing={isMobile ? 1 : 0}>
+                <Grid container item xs={12} spacing={1}>
+                  <Grid item xs={isMobile ? 12 : 6}>
+                    <FormControl
+                      size="small"
+                      fullWidth
+                      error={!!errors.amount && touched.amount}
                     >
-                      {t('Amount')}
-                    </LogFormLabel>
-                    <FastField name="amount">
-                      {({ field, meta }: FieldProps) => (
-                        <Box width="100%">
-                          <FormTextField
-                            {...field}
+                      <LogFormLabel
+                        htmlFor="amount-input"
+                        id="amount-input-label"
+                        required
+                      >
+                        {t('Amount')}
+                      </LogFormLabel>
+                      <FastField name="amount">
+                        {({ field, meta }: FieldProps) => (
+                          <Box width="100%">
+                            <FormTextField
+                              {...field}
+                              size="small"
+                              variant="outlined"
+                              fullWidth
+                              disabled={
+                                pledge?.status === PledgeStatusEnum.Processed
+                              }
+                              type="text"
+                              inputProps={{
+                                'aria-labelledby': 'amount-input-label',
+                              }}
+                              id="amount-input"
+                              error={!!errors.amount && touched.amount}
+                              // eslint-disable-next-line jsx-a11y/no-autofocus
+                              autoFocus
+                            />
+                            <FormHelperText>
+                              {meta.touched && meta.error}
+                            </FormHelperText>
+                          </Box>
+                        )}
+                      </FastField>
+                    </FormControl>
+                  </Grid>
+                  <Grid item xs={isMobile ? 12 : 6}>
```

Comment:
Was there a reason to change these widths? Long currencies like UGS don't fit as well now.